packages/url_launcher/ios/Classes/UrlLauncherPlugin.m

```diff
@@ -144,12 +146,12 @@ - (void)launchURL:(NSString *)urlString
 - (void)launchURLInVC:(NSString *)urlString result:(FlutterResult)result API_AVAILABLE(ios(9.0)) {
   NSURL *url = [NSURL URLWithString:urlString];
   self.currentSession = [[FLTUrlLaunchSession alloc] initWithUrl:url withFlutterResult:result];
-  __weak typeof(self) weakSelf = self;
+  self.currentSession.didFinish = ^(void) {
```

Comment:
Can we do this to match the style of this file?
```objective-c
  __weak typeof(self) weakSelf = self;
  self.currentSession.didFinish = ^(void) {
    weakSelf.currentSession = nil;
  };
```

Reply:
Yes that works just as well!